amazon-cloudwatch-agent-operator.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: amazon-cloudwatch-agent-operator
+  version: 1.6.0
+  epoch: 0
+  description: Software developed to manage the CloudWatch Agent on kubernetes.
+  copyright:
+    - license: Apache-2.0
+
+environment:
```

Comment:
We don't need to specify the environment anymore. `go/build` will take care of it